.github/workflows/tests.yaml

```diff
@@ -0,0 +1,36 @@
+name: "Tests"
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        php: [ '7.1', '7.2', '7.3', '7.4', '8.0' ]
+    name: PHP ${{ matrix.php }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+
+      - name: Install PHP packages
+        run: composer install --no-interaction --prefer-source
```

Comment:
Why do we prefer source?

Reply:
It may not be needed – I've occasionally had problems with components not installing properly without that set.

Reply:
I've removed that and it seems happy